src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,26 +1,24 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
-    # loop through n-1 elements
+def selection_sort(arr):
     for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
-        # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
-        # TO-DO: swap
-
-
-
-
+        current = arr[i]
+        smallest = current
+        smallest_index = i
+        for j in range(i, len(arr)):
+            if (arr[j] < smallest):
+                smallest = arr[j]
+                smallest_index = j
+        arr[i], arr[smallest_index] = smallest, current
     return arr
 
 
-# TO-DO:  implement the Bubble Sort function below
-def bubble_sort( arr ):
-
+def bubble_sort(arr):
+    boolean = True
```

Comment:
so proud to see you using the boolean and while loop. More than a couple people here didn't haha